src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -81,6 +87,27 @@ public void updatePerson(ReadOnlyPerson target, ReadOnlyPerson editedPerson)
         indicateAddressBookChanged();
     }
 
+    /**
+     * Deletes a specified tag from every person in the AddressBook
+     * @param tag
+     * @throws PersonNotFoundException
+     * @throws DuplicatePersonException
```

Comment:
The start of the header comment does not follow the convention. The param and throw tags should have informative comments. Under what condition would `PersonNotFoundException` be thrown so on and forth.
Can `tag` be null?